src/script/E2EIdentity/E2EIdentityVerification.ts

```diff
@@ -81,17 +89,25 @@ export async function getConversationVerificationState(groupId: string) {
 /**
  * Checks if E2EI has active certificate.
  */
-const fetchSelfDeviceIdentity = async (): Promise<WireIdentity | undefined> => {
+const getSelfDeviceIdentity = async (): Promise<WireIdentity | undefined> => {
   const conversationState = container.resolve(ConversationState);
-  const selfMLSConversation = conversationState.getSelfMLSConversation();
-  const userIdentities = await getAllGroupUsersIdentities(selfMLSConversation.groupId);
+
+  // Try to get the self MLS conversation from the conversation state
+  // If the conversation state is not available, try to get the self MLS conversation from backend
+  const selfMLSConversationGroupId =
+    conversationState.selfMLSConversation()?.groupId ||
+    (await getCoreConversationService().getMLSSelfConversation()).group_id;
```

Comment:
```suggestion
    conversationState.selfMLSConversation()?.groupId ??
    (await getCoreConversationService().getMLSSelfConversation()).group_id;
```

Just to be on the safe side :)